### DISCUSSION TITLE
Gitahead

### DISCUSSION THREAD

**Discussion by tayllana** [community/community#114503] [Programming Help] [answered]
Hello! I'm facing an issue: I've been using GitAhead for code versioning on my Ubuntu (Linux) for quite some time. However, last month, I had to reset my computer. When I tried to reinstall GitAhead from the usual website (https://github.com/gitahead/gitahead/releases/tag/v2.7.1), I noticed that there's no longer a version available for Ubuntu. I found some references about using Flatpak to download the application, but unfortunately, the experience was unsatisfactory as the application had many bugs. Could someone please assist me with an effective way to reinstall GitAhead? It can be an older version, not necessarily the latest one. Thank you in advance!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by spenserblack** [comment]
Looking at GitAhead I stumbled on [Gittyup](https://github.com/Murmele/Gittyup), which looks like a maintained continuation of GitAhead. Maybe you would want to check that out?

But if you browse all releases of GitAhead there's [version 2.6.3](https://github.com/gitahead/gitahead/releases/tag/v2.6.3) which seems to have an installation file. It looks like it's a shell script that has an entire gzipped tar file embedded right into it ([`GitAhead-2.6.3.sh`](https://github.com/gitahead/gitahead/releases/download/v2.6.3/GitAhead-2.6.3.sh)). That's a new one for me :sweat_smile: But is that the asset you would have used before to install?

**Reply by tayllana** [reply]
but I already tried to install the .sh file in every way but it didn't work either

**Reply by spenserblack** [reply, marked as answer]
Got it. I just ran the `.sh` file and answered "yes" to the "Do you want to include the subdirectory GitAhead?" The executable's actual location was my `Downloads` folder, which isn't ideal, but it did seem to "install" correctly:

```shell
cd ~/Downloads
chmod +x GitAhead-2.6.3.sh
./GitAhead-2.6.3.sh
# answer prompts
```

That looked like it wrote the `.desktop` file and icons correctly, and it was in my app list.

Could you try that and let me know if you get any errors, or if anything is missing?

**Reply by tayllana** [reply]
It worked very well, thank you very much!!!!!!!!!
I don't know why it didn't work before when I tried
